.github/workflows/ci.yaml

```
@@ -92,4 +92,4 @@ jobs:
   #     run: |
   #       uv pip install build twine
   #       python -m build
-  #       twine upload dist/*
+  #       twine upload dist/*
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)